### ISSUE TITLE
'Esc' no longer closing popups in Safari

### ISSUE THREAD

**Issue by Hildy** [jstanden/cerb#1657]
Client reports, "Pressing escape to close a profile causes the confirmation alert to appear briefly and then disappear again immediately."

Issue has also been noticed on both dev and production instances.

**Comment by jstanden** [comment]
Confirmed this as a Safari issue with a minimal example: https://jsfiddle.net/0feLkbr5/

```html
<html>
  <body>
    <div id="dialog">
      I am a dialog. Press ESC to close me.
    </div>
  </body>
</html>
```

```js
$(document).ready(function() {
  $('#dialog').dialog({
    beforeClose: function(e, ui) {
      return confirm("Are you sure?");
    }
  });
});
```

**Comment by jstanden** [comment]
Fixed in 10.2.6
